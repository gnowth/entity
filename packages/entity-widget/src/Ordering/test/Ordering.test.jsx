import renderer from 'react-test-renderer';
import React from 'react';
import { Fields } from '@entity/core';

import WidgetOrdering from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <WidgetOrdering
      field={new Fields.EnumField()}
      label="test"
      name="test"
      onChange={() => undefined}
      orderingKey="test"
      value=""
    />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
