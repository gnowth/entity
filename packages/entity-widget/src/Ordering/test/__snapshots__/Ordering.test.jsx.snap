// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0.c0 {
  font-size: 1rem;
}

<span
  className=""
  onClick={[Function]}
  onKeyPress={[Function]}
  role="button"
  tabIndex={0}
>
  test
  <i
    className="material-icons sc-EHOje jvzENE c0"
    name="unfold_more"
  >
    unfold_more
  </i>
</span>
`;
